src/main/generated/resources/assets/minecolonies/lang/default.json

```
@@ -2,6 +2,7 @@
   "com.minecolonies.research.civilian.name": "Civilian",
   "com.minecolonies.research.combat.name": "Combat",
   "com.minecolonies.research.technology.name": "Technology",
+  "com.minecolonies.research.achievment.name": "Achievement",
```

```suggestion
  "com.minecolonies.research.achievement.name": "Achievement",
```
for other instances too

---

So, fixed this by fixing the generator and regenning it.